harper-core/src/linting/closed_compounds.rs

```diff
@@ -82,12 +82,94 @@ create_closed_compound_linter!(Northeastern, "north eastern", "northeastern");
 create_closed_compound_linter!(Upholstery, "up holstery", "upholstery");
 create_closed_compound_linter!(Proofread, "proof read", "proofread");
 
+create_closed_compound_linter!(Forever, "for ever", "forever");
+create_closed_compound_linter!(Into, "in to", "into");
```

Comment:
"in to" is not always wrong. Mostly when "in" is part of a phrasal verb or "to" is part of an infinitive verb.